src/index.js

```diff
@@ -39,16 +39,17 @@ export const EDSCEchoform = ({
   useEffect(() => {
     // This effect compares each field for relevancy and validity.
     // valid is true if all of the relevantFields also have valid values.
-    const valid = Object.keys(relevantFields)
+    const relevantInvalidFields = Object.keys(relevantFields.current)
       .filter((key) => {
-        const value = formIsValid[key]
-        const relevant = relevantFields[key]
+        const fieldValid = formIsValid[key]
+        const fieldRelevant = relevantFields.current[key]
 
         // Return only relevant invalid fields
-        return relevant && value === false
-      }).length === 0 // if no fields are returned, the form is valid
+        return fieldRelevant && fieldValid === false
```

Comment:
Could this be `fieldRelevant && !fieldValid`?

Reply:
looks like I can change it if I change the keys I loop through to `formIsValid`, I'll do that in the next PR